app/com/linkedin/drelephant/ElephantRunner.java

```diff
@@ -178,8 +178,11 @@ public void run() {
         result.save();
         long processingTime = System.currentTimeMillis() - analysisStartTimeMillis;
         logger.info(String.format("Analysis of %s took %sms", analysisName, processingTime));
-        MetricsController.setJobProcessingTime(processingTime);
-        MetricsController.markProcessedJobs();
+
+        if(MetricsController.isInitialized()) {
```

Comment:
@shankar37 : This is for the case when MetricsController is turned off. I don't think we have NPE here as _jobProcessingTime, _processedJobs and _skippedJobs are guarded against null check before marking/updating them.